test/jdk/sun/net/www/protocol/https/ChunkedOutputStream.java

```diff
@@ -333,26 +350,48 @@ public static void main(String[] args) throws Exception {
             HttpsURLConnection.setDefaultHostnameVerifier(new NameVerifier());
 
             try {
-                server = new TestHttpsServer(
-                        new ChunkedOutputStream(), 1, 10, loopback, 0);
-                System.out.println("Server started: listening on: " + server.getAuthority());
-                testPlainText(server.getAuthority());
+                // create and initialize a SSLContext
+                KeyStore ks = KeyStore.getInstance("JKS");
+                KeyStore ts = KeyStore.getInstance("JKS");
+                char[] passphrase = "passphrase".toCharArray();
+
+                ks.load(new FileInputStream(System.getProperty("javax.net.ssl.keyStore")), passphrase);
+                ts.load(new FileInputStream(System.getProperty("javax.net.ssl.trustStore")), passphrase);
+
+                KeyManagerFactory kmf = KeyManagerFactory.getInstance("SunX509");
+                kmf.init(ks, passphrase);
+
+                TrustManagerFactory tmf = TrustManagerFactory.getInstance("SunX509");
+                tmf.init(ts);
+
```

Comment:
Could SimpleSSLContext be used here instead of manually writing this code? Same for other tests with same pattern.